spec/models/sample_manifest_spec.rb

```diff
@@ -224,22 +224,9 @@
       context 'library tubes' do
         it 'create 1 tube' do
           # We need to create library tubes as we have downstream dependencies that assume a unique library tube
-          expect { manifest.generate }.to change(LibraryTube, :count).by(count).and change(
-                                                     MultiplexedLibraryTube,
-                                                     :count
-                                                   ).by(0).and change(SampleTube, :count).by(0).and change(
-                                                                                                                                         SampleManifestAsset,
-                                                                                                                                         :count
-                                                                                                                                       )
-                                                                                                                                         .by(
-                                                                                                                                         count
-                                                                                                                                       ).and change(
-                                                                                                                                                                                          BroadcastEvent,
-                                                                                                                                                                                          :count
-                                                                                                                                                                                        )
-                                                                                                                                                                                          .by(
-                                                                                                                                                                                          1
-                                                                                                                                                                                        )
```

Comment:
Oh wow!

Reply:
This change was actually manual. I assumed when I saw it in the diff that prettier or rubocop had done something funky in this set of changes, hadn't realised it was terrible before the last sweep